samples/deprecated/README.md

```diff
@@ -0,0 +1,63 @@
+# Pelorus Samples & Use Cases
+
+## Deploying the Basic-Nginx Sample alongside the Webhook/Generic CommitTime Exporter
+
+1. Fork [this repo](https://github.com/konveyor/pelorus)
+2. Run `oc create namespace pelorus`
+3. Create a GitHub Personal Access Token and store it as a secret in OCP: 
+```
+oc create secret generic github-secret --from-literal=GIT_USER=<username> --from-literal=GIT_TOKEN=<personal access token> --namespace pelorus
+```
+4. Run `helm install operators charts/operators --namespace pelorus`
+5. Update the values.yaml file with the following:
+```
+exporters:
+  instances:
+  - app_name: deploytime-exporter
+    source_context_dir: exporters/
+    extraEnv:
+    - name: APP_FILE
+      value: deploytime/app.py
+    - name: LOG_LEVEL
+      value: DEBUG
+    - name: NAMESPACES
+      value: basic-nginx-prod
+    source_ref: master
+    source_url: https://github.com/konveyor/pelorus.git
+  - app_name: committime-generic
+    env_from_secrets:
+    - github-secret
+    - mongodb-build-webhook
+    source_context_dir: exporters/
+    extraEnv:
+    - name: APP_FILE
+      value: genericexporter/app.py
+    - name: LOG_LEVEL
+      value: DEBUG
+    source_ref: master
+    source_url: https://github.com/konveyor/pelorus.git
+
+webhooks:
+  instances:
+  - app_name: build-webhook
+    source_context_dir: webhooks/
+    extraEnv:
+    - name: APP_FILE
+      value: build-webhook/app.py
```

Comment:
I think you are missing webhooks directory with it's content in this PR ?

Reply:
@weshayutin  I think it was never merged, see https://github.com/konveyor/pelorus/pull/253